db/migrate/20180525135715_fix_note_annotations_index.rb

```diff
@@ -0,0 +1,6 @@
+class FixNoteAnnotationsIndex < ActiveRecord::Migration
+  def change
+    remove_index :note_annotations, :value if index_exists? :note_annotations, :value
```

Comment:
Here is a little gotcha: remove_index is only reversible if a given column option